src/python/platforms/android/logger.py

```diff
@@ -22,8 +23,10 @@
 
 def clear_log():
   """Clear log."""
+  adb.run_as_root()
   adb.run_shell_command(['stop', 'logd'])
   adb.run_shell_command(['start', 'logd'])
+  time.sleep(1)
```

Comment:
Can you check if anything smaller is possible here like 0.5 or 0.1/0.2 in a reliable way ?

Reply:
0.1 works fine, I've ran it for hours and no warnings in logs.